code/game/objects/items/weapons/shields.dm

```diff
@@ -156,6 +156,47 @@
 		set_light(1.5, 1.5, "#006AFF")
 	else
 		set_light(0)
+// tact
+/obj/item/weapon/shield/riot/tact
+	name = "tactical shield"
+	desc = "A highly advanced ballistic shield crafted from durable materials and plated ablative panels. Can be collapsed for mobility."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "tactshield0"
+	force = 3.0
+	throwforce = 3.0
+	throw_speed = 3
+	throw_range = 4
+	w_class = 2
+	attack_verb = list("shoved", "bashed")
+	var/active = 0
+
+/obj/item/weapon/shield/riot/tact/attack_self(mob/living/user)
+	active = !active
+	icon_state = "tactshield[active]"
+	playsound(src.loc, 'sound/weapons/empty.ogg', 50, 1)
+
+	if(active)
+		force = 15
+		throwforce = 5
+		throw_speed = 2
+		w_class = 4
+		slot_flags = SLOT_BACK
+		user << "<span class='notice'>You extend \the [src] downward with a sharp snap of your wrist.</span>"
+	else
+		force = 3
+		throwforce = 3
+		throw_speed = 3
+		w_class = 2
+		slot_flags = null

```

Comment:
This may runtime. `slot_flags = 0` is better, as it'll handle bitwise operations without and danger.